_code/cellPoly/vignettes/CellPoly vignette.Rmd

```diff
@@ -25,7 +25,7 @@ knitr::opts_chunk$set(
 ### Installation
 ```{r installation, eval=FALSE}
 
-#devtools::install_github("https://github.com/Nanostring-Biostats/CosMx-Analysis-Scratch-Space/tree/Main/code/cellPoly")
+#devtools::install_github("Nanostring-Biostats/CosMx-Analysis-Scratch-Space/_code/cellPoly@Main")
 # setwd() in Rstudio to the CellPoly directory
```

Comment:
Can we delete the next two lines (setwd and devtools::install) since we're calling install_github directly?

Reply:
Those commented out lines are there to do another install method. I will make a note (commented lines) of the two install methods.

Reply:
e77fc163c035ceb7db6e4371c7207130624db01a